src/Sil/Bundle/ContactBundle/Resources/views/form/address-form.html.twig

```diff
@@ -0,0 +1,84 @@
+
+<div class="ui card fluid raised">
+    <h1 class="ui icon header">
+        <i class="circular address card icon blue"></i>
+        <div class="content">
+            <div class="sub header">
+                <h3>New Address</h3>
+            </div>
+        </div>
+    </h1>
+    {% if form.vars.value and form.vars.value.id is defined %}
```

Comment:
Should be `{% if form.vars.value is not null and form.vars.value.id is defined %}` ?